kiali/README.md

```diff
@@ -0,0 +1,89 @@
+# Install custom Kiali in Kyma
+
+## Overview
+
+The Kyma Kiali component brings limited configuration options in contrast to the upstream [`kiali-operator`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. The following instructions outline how to achieve such installation in co-existent to the Kyma stack.
+
+## Prerequisites
+
+- Kyma as the target deployment environment
```

Comment:
Do we need any minimum Kyma version?

Reply:
No, should also work with older versions